internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -167,17 +167,33 @@ func (c *cmd) Execute() (statusCode int, err error) {
 		return interfaces.StatusCodeExitWithError, fmt.Errorf("failed to configure Consul access: %v", err)
 	}
 
-	// write a sentinel file to indicate Consul ACL bootstrap is done so that we don't bootstrap ACL again,
-	// this is to avoid re-bootstrapping error and that error can cause the snap crash if restart this process
-	if err := c.writeSentinelFile(); err != nil {
-		return interfaces.StatusCodeExitWithError, fmt.Errorf("failed to write sentinel file: %v", err)
+	// create EdgeX service policy
```

Comment:
So this is where you call `getPolicyByName` first and use that policy if it isn't nil...

Reply:
like I said in the above comments, trying to reduce the complixity of this Execute() func, so it is wrapped in the policy func call

Reply:
I don't agree. You have hidden the `getPolicyByName ` functionality in the create function.